app/services/preserved_object_handler.rb

```diff
@@ -111,17 +157,13 @@ def result_code_msg(response_code, addl=nil)
   end
 
   def result_msg_prefix
-    @msg_prefix ||= "PreservedObjectHandler(#{druid}, #{incoming_version}, #{incoming_size})"
+    @msg_prefix ||= "PreservedObjectHandler(#{druid}, #{incoming_version}, #{incoming_size}, #{storage_dir})"
   end
 
-  # results = [result1, result2]
-  # result1 = {response_code => msg}
-  # result2 = {response_code => msg}
   def log_results(results)
     results.each do |r|
       severity = logger_severity_level(r.keys.first)
       msg = r.values.first
-      # Rails.logger.log(severity, msg, 'PreservedObjectHandler')
       Rails.logger.log(severity, msg)
```

Comment:
so we don't want info about the calling class?  or it's already in the logs?  or christina's logs are capturing it?

Reply:
Just ran a test and ``` Rails.logger.log(severity, msg, 'PreservedObjectHandler')``` logs the same thing as ``` Rails.logger.log(severity, msg)```